rust/operator-binary/src/config/jvm.rs

```diff
@@ -109,20 +80,20 @@ pub fn jvm_config(
             -Djdk.attach.allowAttachSelf=true
             -Djdk.nio.maxCachedBufferSize=2000000
             -Dfile.encoding=UTF-8
-            # Reduce starvation of threads by GClocker, recommend to set about the number of cpu cores (JDK-8192647)
-            -XX:+UnlockDiagnosticVMOptions
-            -XX:GCLockerRetryAllocationCount=32
             # Allow loading dynamic agent used by JOL
             -XX:+EnableDynamicAgentLoading
+            # https://bugs.openjdk.org/browse/JDK-8329528
+            -XX:+UnlockDiagnosticVMOptions
+            -XX:G1NumCollectionsKeepPinned=10000000
 
             -Djavax.net.ssl.trustStore={STACKABLE_CLIENT_TLS_DIR}/truststore.p12
             -Djavax.net.ssl.trustStorePassword={STACKABLE_TLS_STORE_PASSWORD}
             -Djavax.net.ssl.trustStoreType=pkcs12
             -Djava.security.properties={RW_CONFIG_DIR_NAME}/{JVM_SECURITY_PROPERTIES}
             ",
         )),
-        // Copied from https://trino.io/docs/451/installation/deployment.html
-        "451" => Ok(formatdoc!(
+        // Copied from https://trino.io/docs/455/installation/deployment.html#jvm-config
+        "455" => Ok(formatdoc!(
```

Comment:
The config https://trino.io/docs/455/installation/deployment.html#jvm-config is a bit different from https://trino.io/docs/452/installation/deployment.html#jvm-config, did you check that?

Reply:
Sorry the PR wasn't ready for review, still working on it :laughing: that's why I didn't move it to `waiting for review` yet. `supported-versions.adoc` is updated in the commit I'm working on right now.

The config removed the settings I assumed were a workaround for the linked fix. But this fix is available in openjdk-23, which we can't use yet? https://pkgs.org/download/java-23-openjdk-devel but if you have more information let me know.

Reply:
Ok nvm its the `Draft` vs `In review` thing :)
I did not look further into the jvm config, just wanted to point out that something changed.

Reply:
Wasn't sure if I would get a PR number for the changelog with a Draft :D will try it out next time, sorry for the confusion

Reply:
No worries, you did it right, its about whats "in review" :D so my bad...

Reply:
Checked with @sbernauer regarding the jvm settings: https://bugs.openjdk.org/browse/JDK-8330772 ; removed it